cmd/buildah/mount.go

```diff
@@ -21,17 +23,15 @@ var (
 		Usage:          "Mount a working container's root filesystem",
 		Description:    mountDescription,
 		Action:         mountCmd,
-		ArgsUsage:      "CONTAINER-NAME-OR-ID",
+		ArgsUsage:      "CONTAINER-NAME-OR-ID [...]",
```

Comment:
From reading the man page, `buildah-mount` doesn't require arguments.
In that case would `ArgUsage` be `[CONTAINER-NAME-OR-ID, ...]`?

Reply:
`buildah mount` with no arguments results in a list of currently mounted containers, so the container id is indeed optional. Good catch.

Output form this is the same as what @TomSweeneyRedHat is proposing for multiple mount output, so more reason to add that capability.